### PR TITLE
Add link to RFC Guide from jakobo/rfc

### PR DESCRIPTION
**Pull request by spier**
Implements #666. Adding link to https://github.com/jakobo/rfc and description.

## Details

- Add link to repo jakobo/rfc, joining a guide for how to set up a company-internal RFC process.
- Clarifying which 'Requests for Comments' we are referring to here

### CODE REVIEW

**Comment by spier** [comment]
@tsadler1988 you might like this :)

File: patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Important elements of the solution are:
 
 - [Rust][rust] is a good Open Source example of RFC template and process, and has been the basis for many other RFC processes.
 - [Generalized BBC iPlayer & Sounds RFC template](templates/rfc.md), originally based on the [Rust][rust] template
+- [jakobo/rfc](https://github.com/jakobo/rfc) outlines how to set up a company-internal RFC process. It contains a [detailed explanation](https://github.com/jakobo/rfc/blob/master/text/0001-using_rfcs.md) of why RFCs are important and an [RFC template](https://github.com/jakobo/rfc/blob/master/0000-template.md) to help you write your first RFC. It contains contains information such as motivation/rational, guide implementation, a reference implementation, drawbacks, as well as alternatives to the RFC approach. Bonus: The description itself is an RFC, so while reading it you are already getting a sense of how an RFC works.
```
Comment (spier) [inline patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md]:
```suggestion
- [jakobo/rfc](https://github.com/jakobo/rfc) outlines how to set up a company-internal RFC process. It contains a [detailed explanation](https://github.com/jakobo/rfc/blob/master/text/0001-using_rfcs.md) of why RFCs are important and an [RFC template](https://github.com/jakobo/rfc/blob/master/0000-template.md) to help you write your first RFC. It contains information such as motivation/rational, guide implementation, a reference implementation, drawbacks, as well as alternatives to the RFC approach. Bonus: The description itself is an RFC, so while reading it you are already getting a sense of how an RFC works.
```

**Comment by spier** [comment]
@michael-basil as you were reading our patterns recently anyways, would you mind taking a look at this change here? It is a fairly small change, so I am just looking for a 2nd pair of eyes to see if the text is easy to understand and if this is an improvement to the pattern.

If you don't have time, no worries!

**Comment by michael-basil** [comment]
Taking a look now.

**Review by michael-basil — COMMENTED** [review]
@spier - After reading through the additional language and the attached links I believe it is an improvement to the pattern.

The links to the company-internal RFC process are opinionated helping to form a more concrete idea of how this could play out.  There are some baked in biases and assumptions, but this is okay and serves the purpose well.

If it were me on the other side, with the level of understanding I currently have, I would approve this once the automated check failure is cleared.

**Comment by spier** [comment]
> @spier - After reading through the additional language and the attached links I believe it is an improvement to the pattern.

Great to hear! Thank you for the review!

> 
> The links to the company-internal RFC process are opinionated helping to form a more concrete idea of how this could play out. There are some baked in biases and assumptions, but this is okay and serves the purpose well.
> 
> If it were me on the other side, with the level of understanding I currently have, I would approve this once the automated check failure is cleared.

Thanks for reminding me of the issues with the vale check. I had mistakenly assumed that this was a false positive, however now I see that there is a systematic issues with our check, as it does not find out custom word lists at all. I am debugging this now.

This does not have to stop this PR from going live though. Will make fixes in a separate PR, once I figure out what's up there :)

Thanks again @michael-basil !

**Pull request merged**